### ISSUE TITLE
undefined: syscall.Win32FileAttributeData

### ISSUE THREAD

**Issue by halorium** [pkg/sftp#165]
I am getting the following error:
```
# github.com/pkg/sftp
../../../../pkg/sftp/request-stub.go:8: undefined: syscall.Win32FileAttributeData
```

Which I believe is related to commit: 1189af57876f9ce079097fe9f33a7cd9488df41d

I build with the following command:
```
GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -i -ldflags '-s -w'
```

#### System details

```
go version go1.8 darwin/amd64
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/Loren/go"
GORACE=""
GOROOT="/usr/local/Cellar/go/1.8/libexec"
GOTOOLDIR="/usr/local/Cellar/go/1.8/libexec/pkg/tool/darwin_amd64"
GCCGO="gccgo"
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/ly/kvvdk7ps6cxfsj9dxvd69p7r0000gn/T/go-build726286195=/tmp/go-build -gno-record-gcc-switches -fno-common"
CXX="clang++"
CGO_ENABLED="1"
PKG_CONFIG="pkg-config"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
GOROOT/bin/go version: go version go1.8 darwin/amd64
GOROOT/bin/go tool compile -V: compile version go1.8 X:framepointer
uname -v: Darwin Kernel Version 15.6.0: Mon Aug 29 20:21:34 PDT 2016; root:xnu-3248.60.11~1/RELEASE_X86_64
ProductName:	Mac OS X
ProductVersion:	10.11.6
BuildVersion:	15G1004
lldb --version: lldb-360.1.50
```

**Comment by CameronGo** [comment]
I'm seeing this exact issue as well.  As far as I can tell it started today.  Since we always clear GOPATH in our build scripts, it seems likely the commit mentioned above is indeed responsible.

**Comment by jcaberio** [comment]
also experiencing this error.

**Comment by eikenb** [comment]
This should be fixed now (as of commit 1189af57876f9ce079097fe9f33a7cd9488df41d). 
Sorry for the breakage. Trying to figure out how to support windows when not having access to it is being a pain.
